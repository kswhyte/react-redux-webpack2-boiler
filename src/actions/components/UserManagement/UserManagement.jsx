import React from 'react';

import './user-management.css';

const UserManagement = () => {
    return (
        <div className="user-management-block">
            <h1>User Management</h1>
        </div>
    )
}

UserManagement.propTypes = {};

export default UserManagement;
