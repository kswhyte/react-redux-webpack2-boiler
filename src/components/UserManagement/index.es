import Component from './UserManagement';
export default Component;