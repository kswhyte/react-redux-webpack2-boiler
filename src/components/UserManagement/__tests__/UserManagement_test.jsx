import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import UserManagement from '../UserManagement';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('UserManagement', module);

stories.add('Search Patient Story', () => {
    const userManagementStory = (
      <UserManagement/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the UserManagement component without crashing', () => {
            let output = shallow(userManagementStory);
            expect(output.find('.user-management-block').length).toEqual(1);
        });
    }));
    return userManagementStory;
});
