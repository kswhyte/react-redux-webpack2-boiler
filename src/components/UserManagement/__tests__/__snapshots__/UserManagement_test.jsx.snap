// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Patient Story 1`] = `
<div
  className="user-management-block"
>
  <h1>
    User Management
  </h1>
</div>
`;
